drivers/planar/codes.txt

```diff
@@ -0,0 +1,17 @@
+op ** display.power = on             // Power ON **check this
```

Comment:
Try to keep any notes like this, or other items not specifically relating to the driver out of this repo.